iOSClient/Extensions/UIAlertController+Extension.swift

```diff
@@ -87,6 +92,42 @@ extension UIAlertController {
             textField.isSecureTextEntry = true
             textField.placeholder = NSLocalizedString("_password_", comment: "")
         }, completion: completion)
-
+        
+    }
+    
+    
+    static func createEncryptedFolder(serverUrl: String, urlBase: NCUserBaseUrl,fileNameFolder:String, completion: (_ error: NKError) -> Void){
```

Comment:
Add Comment description for this function

Reply:
Fixed the required changes.